services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/util/NetworkConfigManagerConstants.java

```diff
@@ -0,0 +1,5 @@
+package com.futurewei.alcor.netwconfigmanager.util;
+
+public class NetworkConfigManagerConstants {
```

Comment:
Regarding the naming, I would recommend to make it short, for example, Constants. 

The full class name include NetworkConfigManager in the path.

Reply:
You mean, rename the class to Constants? I think that makes sense. I will address in the next commit.

Reply:
Right.